### DISCUSSION TITLE
DRC error

### DISCUSSION THREAD

**Discussion by SMSajadi99** [gdsfactory/gdsfactory#3039] [Q&A]
hello , I hope you are fine
I have a problem with this part and I can't execute this part correctly, maybe you can help me

![image](https://github.com/user-attachments/assets/158fac40-4c78-403f-847f-5c66efd7a0b4)

![image](https://github.com/user-attachments/assets/8dcd79b7-2f78-4041-b1df-556baa531712)

![image](https://github.com/user-attachments/assets/5edb01d2-8df1-4732-9725-9c5fc19be756)

**Comment by sebastian-goeldi** [comment]
Layers will probably expect a tuple vs the gdsfactory internal functions. You can cast layers to tupe with `tuple(LAYER)` if they are one of the gdsfactory LayerEnums or else you can use 

```python
li = gf.kcl.get_info(LAYER)
layer_tuple = (li.layer, li.datatype)
```

This will get the information from the layout object (It will get a `gf.kdb.LayerInfo` object describing the layer)
